js/app.js

```diff
@@ -44,55 +19,21 @@ window.addEventListener('DOMContentLoaded', function() {
   var messageArea = document.getElementById('message-container');
 
   function enableImport() {
-    var timeEnable;
-    if (!accessToken || !tokenValidity) {
-      timeEnable = -1
-    } else {
-      timeEnable = tokenValidity - Date.now();
-    }
-
-    if (timeEnable > 0) {
-      importButton.style.display = '';
-      authorizeButton.querySelector('div[data-l10n-id]').dataset.l10nId =
-        'reauthorize';
+    importButton.style.display = '';
+    authorizeButton.querySelector('div[data-l10n-id]').dataset.l10nId =
+      'reauthorize';
+    navigator.mozL10n.ready(() => {
       navigator.mozL10n.translate(authorizeButton);
-      setTimeout(disableImport, timeEnable);
-    } else {
-      disableImport();
-    }
+    });
```

Comment:
Setting the data-l10n-id in the JS feels wrong, and it makes the initialization really painful. So yeah, I’d prefer two elements (or two spans within the same button + a class, whatever works).